terminus-ui/src/confirmation/confirmation.directive.spec.ts

```diff
@@ -205,4 +212,42 @@ describe(`TsConfirmationDirective`, () => {
     });
   });
 
+  describe(`Custom Button Text`, () => {
+
+    test(`confirmation button should accept custom text`, () => {
```

Comment:
All `test` statements should follow the sentence format `should _____`. You can poke around other spec files for inspiration